main/resource-access.bs

```diff
@@ -142,19 +139,151 @@ through POST, using the HTTP `Slug` header as defined in [[!RFC5023]].
+
+When using Web Access Control ([[#wac]]):
+
+Servers MUST indicate their support for HTTP Methods by responding to HTTP
+`GET`, `HEAD` and `OPTIONS` requests on the resource's URL with the HTTP
+Method tokens in the HTTP response header `Allow`.
+
+Servers MUST indicate supported media types in the `Accept-Patch` [[!RFC5789]]
+and the `Accept-Post` [[!LDP]] headers for matching HTTP methods in `Allow`
+header value.
+
+Servers MAY include the HTTP `Accept-Patch` and `Accept-Post` headers in the
+response of a `OPTIONS *` request.
+
+[[Source](https://github.com/solid/specification/issues/85#issuecomment-575386251)]
+[[Source](https://github.com/solid/specification/issues/43)]
+
+### Writing resources ### {#write}
+
+Servers MUST support the HTTP `PUT`, `POST` and `PATCH` methods [[!RFC7231]]
+for clients to create and update resources.
+[[Source](https://github.com/solid/specification/issues/39#issuecomment-538017667)]
+
+Servers MUST create intermediate containers and include corresponding
+containment triples in container representations derived from the URI path
+component of `PUT`, `POST` and `PATCH` requests.
+[[Source](https://github.com/solid/specification/issues/68#issuecomment-561690124)]
+
+Servers MUST allow creating new resources with a `POST` request to URI path
 ending `/`. Servers MUST create a resource with URI path ending `/{id}` in
 container `/`. Servers MUST create a container with URI path ending `/{id}/`
 in container `/` for requests including the HTTP `Link` header with
 `rel="type"` targeting a valid LDP container type. Servers MUST handle
 subsequent requests to the newly created container's URI as if it is a valid
-LDP container type by including it the HTTP response's `Link` header.
+LDP container type by including the HTTP response's `Link` header.
+[[Source](https://github.com/solid/specification/pull/160#issuecomment-636822687)]
+
+For resource representations having an algorithm to perform an append
+operation, servers MUST update resources for HTTP `POST` requests. Servers
+MUST perform RDF merge [[!RDF11-MT]] on RDF sources [[!RDF-CONCEPTS]].
+[[Source](https://github.com/solid/specification/issues/108#issuecomment-573655070)]
+
+When a `PUT` or `PATCH` method request targets an auxiliary resource, the
+server MUST create or update it. When a `POST` method request with the `Slug`
+header targets an auxiliary resource, the server `MUST` respond with the `403`
+status code and response body describing the error.
+[[Source](https://github.com/solid/specification/issues/42#issuecomment-616688848)]
+
+Servers SHOULD NOT allow HTTP `POST`, `PUT` and `PATCH` to update a
+container's containment triples; if the server receives such a request, it
+SHOULD respond with a `409` status code.
+[[Source](https://github.com/solid/specification/issues/40#issuecomment-573358652)]
+
+Clients MUST use the HTTP `If-None-Match` header with a value of `"*"` to
```

Comment:
Servers refuse if they don't?

Reply:
The spec currently requires servers to implement 7232 - rough consensus in https://github.com/solid/specification/issues/33 seems to keep it that way - and there are no additional requirements as far as I can tell from the issues.